### ISSUE TITLE
Question: Is there a simple way to localize / translate strings?

### ISSUE THREAD

**Issue by SomwareHR** [Splode/pomotroid#253]
Hi, 

For me, this is the best Pomodoro program :) Simple and effective! Great!

I am not much into Electron projects, I can see strings are in several files. 
Is there a simple way I can localize Pomotroid?

Thanks!

**Comment by Splode** [comment]
@SomwareHR Yes, Pomotroid now supports localization with 8 languages currently supported. I've added a section in [the contributing document](https://github.com/Splode/pomotroid/blob/main/CONTRIBUTING.md) on how additional languages can be added.
